### ISSUE TITLE
Sorting Error on Configure Fields Settings Page

### ISSUE THREAD

**Issue by mattwardcc** [weareferal/craft-matrix-field-preview#95]
I think there might be an issue with the `Configure Fields` option when first installing the plugin. I just installed it on a site I'm developing to test for the first time, and when I visit the `Configure Field` option in the settings, I'm getting the following error:

`Getting unknown property: weareferal\matrixfieldpreview\records\MatrixFieldConfigRecord::sortOrder`

Following the stack trace, the main error appears to be thrown `in /Users/matt.ward/Server/craft-and-crew-v3/vendor/weareferal/matrix-field-preview/src/services/BaseFieldConfigService.php at line 41– [yii\db\BaseActiveRecord::__get](http://www.yiiframework.com/doc-2.0/yii-db-baseactiverecord.html#__get()-detail)('sortOrder')`

The specific block of code that appears to be throwing the error identified as:

```
if ($sort) {
    usort($fieldConfigs, function ($a, $b) {
        if ($a->sortOrder === $b->sortOrder) {
             return 0;
        }
        return $a->sortOrder > $b->sortOrder ? 1 : -1;
    });
}
```             

I ran a simple test locally by adding the following at the beginning of the usort function:

```
if ( ! isset($a->sortOrder) || ! isset($b->sortOrder) ) {
    return 1;
}
```

Basically, I just wanted to see what would happen if I quickly checked for the existence of the sortOrder property on the sort objects. This removed the error and the settings screen loaded for me. I've since removed the test code in my vendor folder and I did not proceed with setting any functions because I'm not sure what other ramifications the change might have had.

Looking at the change history in this repo, however, I did notice that this exact bit of code seemed to have been updated in the last release to resolve issue #93, so I'm wondering if that's where this potential issue may have been introduced? 

We have a strong interest in implementing this plugin on projects to improve the UX of adding modules :)

Cheers and thanks,

Matt

**Comment by ljclark** [comment]
Also experiencing this same issue.

Plugin version: v4.0.3 - everything else works fine, but can't access the 'Configure Fields' page.

PHP version | 8.0.22
Craft edition & version | Craft Pro 4.2.3

**Comment by timmyomahony** [comment]
Yep I think I know what's going on here. I'll get a release out as soon as I can to address.

**Comment by timmyomahony** [comment]
(Thanks for reporting btw)

**Comment by timmyomahony** [comment]
This should now be fixed in the latest version `4.0.4`. Let me know if it fixes the issue
